connect-location/src/main/java/com/ifttt/location/AwarenessGeofenceProvider.java

```diff
@@ -157,29 +171,37 @@ static void diffFences(
                 LocationFieldValue region = userFeatureField.value;
                 switch (userFeatureField.fieldType) {
                     case FIELD_TYPE_LOCATION_ENTER:
-                        callback.onAddFence(id,
+                        callback.onAddFence(
+                            id,
+                            region,
                             LocationFence.entering(region.lat, region.lng, region.radius),
                             enterPendingIntent
                         );
                         fenceKeysToRemove.remove(id);
                         break;
                     case FIELD_TYPE_LOCATION_EXIT:
-                        callback.onAddFence(id,
+                        callback.onAddFence(
+                            id,
+                            region,
                             LocationFence.exiting(region.lat, region.lng, region.radius),
                             exitPendingIntent
                         );
                         fenceKeysToRemove.remove(id);
                         break;
                     case FIELD_TYPE_LOCATION_ENTER_EXIT:
                         String enterFenceKey = getEnterFenceKey(id);
-                        callback.onAddFence(enterFenceKey,
+                        callback.onAddFence(
+                            enterFenceKey,
```

Comment:
can use `getEnterFenceKey(id)` directly here instead of storing it as a variable first

Reply:
We do use this variable twice in this same block.